dependencyManagement/build.gradle.kts

```diff
@@ -115,7 +116,7 @@ val DEPENDENCIES = listOf(
   "org.spockframework:spock-junit4:2.0-groovy-2.5",
   "org.scala-lang:scala-library:2.11.12",
   "org.springframework.boot:spring-boot-dependencies:2.3.1.RELEASE",
-  "io.grpc:grpc-netty-shaded:${grpcVersion}"
+  "io.grpc:grpc-netty-shaded:${grpcVersion}",
```

Comment:
```suggestion
  "io.grpc:grpc-netty-shaded:${grpcVersion}"
```

Reply:
I think code formatting added this, or something, didn't touch this line. It's good that Kotlin supports this trailing comma now right?